<?xml version="1.0"?>
<launch>
  <arg name="tf_prefix" default=""/>

  <group if="false">
  <include file="$(find bullet_server)/launch/bullet_server.launch">
    <arg name="use_rviz" value="false"/>
    <arg name="tf_prefix" value="$(arg tf_prefix)"/>
  </include>
  </group>
  <node pkg="bullet_server" type="ground_plane.py" name="ground_plane"
      output="screen"
      if="false">
  </node>

  <node pkg="bullet_server" type="tracks.py" name="tracked_vehicle"
      output="screen">
  </node>
  <node pkg="rostopic" type="rostopic" name="back_left"
    args="pub wheel_motor_0_0/target_ang_motor_vel std_msgs/Float64 0.7" output="screen"/>
  <node pkg="rostopic" type="rostopic" name="front_left"
    args="pub wheel_motor_0_1/target_ang_motor_vel std_msgs/Float64 0.7" output="screen"/>
  <node pkg="rostopic" type="rostopic" name="back_right"
    args="pub wheel_motor_1_0/target_ang_motor_vel std_msgs/Float64 1.1" output="screen"/>
  <node pkg="rostopic" type="rostopic" name="front_right"
    args="pub wheel_motor_1_1/target_ang_motor_vel std_msgs/Float64 1.1" output="screen"/>

</launch>
